### ISSUE TITLE
should "require-input-label" apply to other elements too?

### ISSUE THREAD

**Issue by zelaznik** [ember-template-lint/ember-template-lint#1530]
# Summary:

I'm using axe-core as the gold standard:

https://dequeuniversity.com/rules/axe/3.5/label

It specifically says:

```
Form elements that should have labels
  - Text entry fields, e.g. <input type="text">, <input type="password"> and <textarea>
  - Radio buttons, <input type="radio">
  - Checkboxes, <input type="checkbox">
  - Select menus, <select>
```

I noticed this discrepancy when I was finding some axe errors that didn't raise lint errors.

# Possible implementations:

  - Include the `<textarea>` and `<select>` tags in this rule.
  - Create parallel rules for these components

If you think this is a worthwhile enhancement, let me know.  I'll put a PR together.

**Comment by zelaznik** [comment]
https://github.com/ember-template-lint/ember-template-lint/pull/1549
